test/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/SubsidyControllerSpec.scala

```diff
@@ -732,6 +732,10 @@ class SubsidyControllerSpec
               val input = doc.getElementById(elementId).attributes().get("value")
               input shouldBe subsidyJourney.claimAmount.value.map(_.amount).getOrElse("")
 
+              doc.getElementById("currency-code-hint").text() shouldBe messageFromMessageKey(
+                "add-claim-amount.input-hint-payment"
```

Comment:
please use the actual message instead of the message key. `If the payment amount was in pounds, it will be converted into euros.`